api/query/atoms.go

```diff
@@ -575,5 +626,10 @@ func unmarshalQ(b []byte) (AbstractQuery, error) {
 	if err == nil {
 		return s, nil
 	}
+	var c AbstractCount
+	err = json.Unmarshal(b, &c)
+	if err == nil {
+		return c, nil
+	}
 	return nil, errors.New(`Failed to parse query fragment as test name pattern, test status constraint, negation, disjunction, or conjunction`)
```

Comment:
nit: `Failed to parse query fragment as test name pattern, test status constraint, negation, disjunction, conjunction, sequential or count`

Reply:
FYI the little +/- icon 
![image](https://user-images.githubusercontent.com/777956/54161152-b30e8f80-4427-11e9-8875-20d30261bb32.png)

Lets you make a suggestion I can apply with the click of a button.

```suggestion
	return nil, errors.New(`Failed to parse query fragment as test name pattern, test status constraint, negation, disjunction, conjunction, sequential or count`)
```